.markdownlinkcheck.json

```
@@ -3,6 +3,12 @@
     {
       "pattern": "^https://github.com/\\S+/\\S+/(issues|pull)/[0-9]+"
     },
+    {
+      "pattern": "^https://github.com/\\S+/\\S+/actions"
+    },
+    {
+      "pattern": "^https://github.com/settings/tokens"
```

```suggestion
      "pattern": "^https://github.com/settings"
```

Wouldn't this make more sense?